CMakeLists.txt

```diff
@@ -7,8 +7,25 @@ if(NOT DEFINED SimpleITKPythonPackage_SUPERBUILD)
   set(SimpleITKPythonPackage_SUPERBUILD 1)
 endif()
 
+# Set a default build type if none was specified
+if(NOT CMAKE_BUILD_TYPE AND NOT CMAKE_CONFIGURATION_TYPES)
+  message(STATUS "Setting build type to 'Release' as none was specified.")
+  set(CMAKE_BUILD_TYPE Release CACHE STRING "Choose the type of build." FORCE)
+  mark_as_advanced(CMAKE_BUILD_TYPE)
+  # Set the possible values of build type for cmake-gui
+  set_property(CACHE CMAKE_BUILD_TYPE PROPERTY STRINGS "Debug" "Release"
+    "MinSizeRel" "RelWithDebInfo")
```

Comment:
Currently the project has the languages set to `NONE`.  If this was C++ would we not get these default properties?